src/main/java/com/google/sps/data/QuestionClass.java

```diff
@@ -0,0 +1,55 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+public final class QuestionClass{
+  /*Class that creates a question */
+  private final String question;
+  private final long questionID;
+  private final double marks;
+  private final String ownerID;
+
+  public QuestionClass(String question, long questionID, double marks, String ownerID)
```

Comment:
Is there no existing ORM for datastore?

Reply:
Yes, we will be removing unused classes.